hypercertContribution.json

```diff
@@ -0,0 +1,54 @@
+{
+  "lexicon": 1,
+  "id": "app.certified.hypercert.contribution",
+  "defs": {
+    "main": {
+      "type": "record",
+      "description": "A contribution made toward a hypercert's impact.",
+      "key": "any",
+      "record": {
+        "type": "object",
+        "required": ["contributor", "role", "createdAt"],
+        "properties": {
+          "hypercert": {
+            "type": "ref",
+            "ref": "app.certified.hypercert.record",
+            "description": "Reference to the hypercert"
```

Comment:
After some research I'm not sure this would result in a reference - I _think_ it actually embeds a full hypercert record inside this lexicon, which is not what we want.  In that case, we need this to be a `com.atproto.repo.strongRef` due to ATProto not supporting links to specific record types (see https://github.com/bluesky-social/atproto/discussions/3661).